packages/reactive-dict/reactive-dict.js

```diff
@@ -164,6 +164,24 @@ _.extend(ReactiveDict.prototype, {
     
   },
 
+  delete: function(key) {
+    var self    = this;
+    var oldKeys = self.keys;
+
+    _.each(oldKeys, function(value, oldKey) {
+      if (key == oldKey){
+        changed(self.keyDeps[oldKey]);
+        changed(self.keyValueDeps[oldKey][value]);
+        changed(self.keyValueDeps[oldKey]['undefined']);
+        delete self.keys[oldKey]; // clean up
+      }
+    });
+
+    // trigger a change regardless if any matching key found...
+    self.allDeps.changed();
+
```

Comment:
@stubailo I totally agree, I don't have strong feelings either way.

I mean, that feels like the technically right way to handle it, but for users/devs, I can picture situations where a user would delete a key, and expect a rerun. 

Let's start with the more performant way (only when changed), and if anyone makes a feature request for always triggering reactive updates it could be handle via an option...

@glasser any thoughts?